src/App.css

```diff
@@ -290,162 +290,208 @@ footer .footerin .w30 {
+	margin: 0 auto;
+	padding: 0;
+	display: block;
+	background-color: #f0f9fe;
+	box-shadow: inset 0 0 10px -5px #f0f9fe;
 }
 
 #search .search-in .results .header h2 {
-  font-size: 18px;
-  font-weight: bold;
-  margin: 0;
-  padding: 20px;
-  color: #2f97f2;
+	font-size: 18px;
+	font-weight: bold;
+	margin: 0;
+	padding: 20px;
+	color: #2f97f2;
 }
 
 #search .search-in .results .content {
-  width: 100%;
-  height: 537px;
-  overflow: scroll;
+	width: 100%;
+	height: 537px;
+	overflow: scroll;
 }
 
 #search .search-in .results .content .location {
-  width: calc(100% - 40px);
-  height: auto;
-  margin: 20px auto;
-  padding: 0;
-  background-color: #f0f9fe;
-  cursor: pointer;
-  border-radius: 3px;
-  box-shadow: -10px 10px 20px -11px #bdbdbd;
+	width: calc(100% - 40px);
+	height: auto;
+	margin: 20px auto;
+	padding: 0;
+	background-color: #f0f9fe;
+	cursor: pointer;
+	border-radius: 3px;
+	box-shadow: -10px 10px 20px -11px #bdbdbd;
+	overflow: hidden;
 }
 
 #search .search-in .results .content .location h2 {
-  font-size: 20px;
-  font-weight: bold;
-  display: block;
-  margin: 0;
-  padding: 20px;
-  color: #2f97f2;
-  position: relative;
+	font-size: 20px;
+	font-weight: bold;
+	display: block;
+	margin: 0;
+	padding: 20px;
+	color: #2f97f2;
+	position: relative;
 }
 
 #search .search-in .results .content .location h2 span {
-  position: absolute;
-  right: 18px;
-  top: 24px;
-  font-size: 14px;
-  letter-spacing: 1px;
-  color: #151515;
+	position: absolute;
+	right: 18px;
+	top: 24px;
+	font-size: 14px;
+	letter-spacing: 1px;
+	color: #151515;
 }
 
 #search .search-in .results .content .location address,
 #search .search-in .results .content .location p {
-  font-size: 16px;
-  font-weight: normal;
-  font-style: normal;
-  color: #222;
-  padding: 0 15px 15px 50px;
-  margin: 0;
-  line-height: 30px;
-  position: relative;
+	font-size: 16px;
+	font-weight: normal;
+	font-style: normal;
+	color: #222;
+	padding: 0 15px 15px 50px;
+	margin: 0;
+	line-height: 30px;
+	position: relative;
 }
 
 #search .search-in .results .content .location p {
-  padding: 10px 20px 27px 20px;
+	padding: 10px 20px 27px 20px;
 }
 
 #search .search-in .results .content .location address i,
 #search .search-in .results .content .location p i {
-  font-size: 20px;
-  color: #2f97f2;
-  margin-right: 10px;
+	font-size: 20px;
+	color: #2f97f2;
+	margin-right: 10px;
 }
 
 #search .search-in .results .content .location address i {
-  margin-right: 15px;
-  position: absolute;
-  left: 20px;
-  top: 4px;
+	margin-right: 15px;
+	position: absolute;
+	left: 20px;
+	top: 4px;
 }
 
 #search .maparea {
-  width: 60%;
-  height: 100%;
-  margin: 0;
-  padding: 0;
-  position: relative;
+	width: 60%;
+	height: 100%;
+	margin: 0;
+	padding: 0;
+	position: relative;
+}
+
+#search .uberBtn {
+	position: relative;
+	display: block;
+	background: black;
+	color: white;
+	border-radius: 3px;
+	padding: 8px;
+	border: none;
+	float: right;
+	margin: 0px 15px 15px;
```

Comment:
it shud be 0 15px 15px https://stylelint.io/user-guide/rules/length-zero-no-unit